package.json

```diff
@@ -63,7 +63,7 @@
       "prettier --write"
     ],
     "*.robot": [
-      "robotidy --config tests/robot-tests/robotidy.toml"
+      "python3 -m pipenv run robotidy --config tests/robot-tests/robotidy.toml tests/robot-tests/tests/"
```

Comment:
You don't need to pass it a path here. This would be quite inefficient if you had to format the entire test directory even if you only changed 1 file.